docs/source/api_ref_data.rst

```diff
@@ -37,18 +33,6 @@ Types
     Message
     Role
 
-Converters
-----------
-
-Converts data from common JSON formats into a torchtune :class:`Message`.
-
-.. autosummary::
-    :toctree: generated/
-    :nosignatures:
-
-    get_sharegpt_messages
-    get_openai_messages
-
 .. _message_transforms_ref:
 
 Message transforms
```

Comment:
nit: Can we call these `ToMessage` transforms to convey that they convey immediately that they convert data to message format?

Reply:
I've used "Message transforms" throughout the docs, so I'll leave updating all those references for a future PR and keep this as is